cmd/cstor-pool-mgmt/pool/v1alpha2/import.go

```diff
@@ -0,0 +1,54 @@
+package v1alpha2
+
+import (
+	"github.com/golang/glog"
+	"github.com/openebs/maya/cmd/cstor-pool-mgmt/controller/common"
+	"github.com/openebs/maya/cmd/cstor-pool-mgmt/volumereplica"
+	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+	api "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha2"
+	zfs "github.com/openebs/maya/pkg/zfs/cmd/v1alpha1"
+)
+
+// Import will import pool for given CSP object.
+// It will also set `cachefile` property for that pool
+// if it is mentioned in object
+func Import(csp *api.CStorNPool) (string, bool, error) {
+	var err error
+
+	if poolExist := checkIfPoolPresent(PoolName(csp)); poolExist {
+		return "", true, nil
```

Comment:
returning true here doesn't handle the case where in previous iteration GetVolumes failed and status set to Failed..
however, revisiting this code is required

Reply:
This need to be handled according to volume-replica-controller. I kept it as it is from old `cstor-pool-mgmt` controller.